app/src/main/java/fr/neamar/kiss/pojo/AppPojo.java

```diff
@@ -11,19 +11,39 @@ public static String getComponentName(String packageName, String activityName,
     public final String packageName;
     public final String activityName;
     public final UserHandle userHandle;
-    public boolean excluded;
+
+    private boolean excluded;
+    private boolean excludedFromHistory;
 
     public AppPojo(String id, String packageName, String activityName, UserHandle userHandle,
-                   boolean isExcluded) {
+                   boolean isExcluded, boolean isExcludedFromHistory) {
         super(id);
 
         this.packageName = packageName;
         this.activityName = activityName;
         this.userHandle = userHandle;
+
         this.excluded = isExcluded;
+        this.excludedFromHistory = isExcludedFromHistory;
     }
 
     public String getComponentName() {
         return getComponentName(packageName, activityName, userHandle);
     }
+
+    public boolean isExcluded() {
```

Comment:
Ok, I know I'm going against the general Java flow here but why add 4 methods when you can just make it public without changing anything?

I assume I'm missing something because everyone seems to do this in Java, but to me it just looks like boilerplate code + added overhead on the function stack.

Reply:
> Ok, I know I'm going against the general Java flow here but why add 4 methods when you can just make it public without changing anything?

So that someone that hasn't even seen the file can know that the values can change at any time :)